lnrpc/walletrpc/walletkit_server_test.go

```diff
@@ -0,0 +1,59 @@
+//go:build walletrpc
+// +build walletrpc
+
+package walletrpc
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/stretchr/testify/require"
+)
+
+// TestWitnessTypeMappings tests that the mapping between the witness type
+// enum defined in the `input` package and the witness type enum protocol
+// buffer definition in the `walletrpc` package is correct.
+func TestWitnessTypeMapping(t *testing.T) {
+	t.Parallel()
+
+	require.Equal(
+		t,
+		len(allWitnessTypes),
+		len(WitnessType_name)-1, // don't count default value
+		"number of witness types should match proto definition",
+	)
+
+	// test that for each individual witness type mapping 1) the mapped
+	// witness types have the same numerical value, and 2) the string
```

Comment:
I find "individual witness type mapping" a bit complicated maybe just write: tests that both enum types map to the same numerical value and their string representation is equivalent ?

Reply:
agreed - simplified it